test/testdrive/patch

```diff
@@ -0,0 +1,1429 @@
+commit dc1d1ad0dd517e3e8636bda97fef470ba5fdfecb
```

Comment:
You might not want to commit this file 😀

Reply:
I removed the file.

Reply:
Thank you.